src/Symfony/Component/HtmlSanitizer/Tests/HtmlSanitizerCustomTest.php

```diff
@@ -0,0 +1,404 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HtmlSanitizer\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HtmlSanitizer\HtmlSanitizer;
+use Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig;
+use Symfony\Component\HtmlSanitizer\Visitor\AttributeSanitizer\AttributeSanitizerInterface;
+
+class HtmlSanitizerCustomTest extends TestCase
```

Comment:
Final please

Reply:
Nope, we don't use `final` on test cases, for the same reasons we don't use `void` either.

Reply:
Ok, cannot check right now, but as far as I remember, we do use it for notifier tests